sites/fast-website/src/app/components/card-section/index.ts

```diff
@@ -0,0 +1,14 @@
+import { customElement, FASTElement } from "@microsoft/fast-element";
+import { CardSectionTemplate as template } from "./card-section.template";
+import { CardSectionStyles as styles } from "./card-section.styles";
+import { FASTCard } from "@microsoft/fast-components";
+
+@customElement({
+    name: "card-section",
+    template,
+    styles,
+})
+export class FeatureCard extends FASTElement {}
```

Comment:
I probably would remove FAST as part of the imported name and just go with Element or Card.  The imported class is already fast, so it just simplifies slightly.  Unless there is extra meaning that's necessary.  This is just my opinion.

Reply:
Just for context, FASTElement is the name of the class which is being extended. It does appear that FASTCard is not being used here though - if it's not, can we drop that import?

Reply:
That's a great point.  Let's make sure that there are no unused imports, variables, or even compiler warnings.

Reply:
Excellent point.  I will update that.  I actually meant to extend FASTCard as it is, so that needed a change anyway.